SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js

```diff
@@ -0,0 +1,35 @@
+define(['../module'], function (module) {
+    'use strict'
+
+    module.service('$appVersionService', function (
+    ) {
+        let appInfo = {revision:'',version:'',splunk_version:''};
+        let documentationAppVersion='';
+        const getAppVersion = () => {
+            return appInfo;
+        }
+
+        /**
+         * Generates and returns the browser base URL + Splunk Port
+         */
```

Comment:
Add the correct documentation to the function. That description doesn't apply to the function